### ISSUE TITLE
Switch Mod / SMod colors

### ISSUE THREAD

**Issue by RethoughtSolutions** [RethoughtSolutions/LPP-Website-Issues#1]
In genereal, the darker a color is, the stronger it feels.
So it only makes sense to make SMods color dark blue and Mod color light blue.

Also, I would like to question having SMods.
I think having Sup and Mod is enough as role. However, that decision is up to you.

If we will have all 3 I suggest the following:
Admin - Red
SMod - Black
Mod - Dark Blue
Supp - Light Blue

**Comment by ViridianLPP** [comment]
I agree with this; the color switch I would deem quite handy because, well, yeah. What Rethought said. And the addition of a Super Mod... I dunno, it could be done.
